runner/settings.go

```diff
@@ -22,6 +22,7 @@ type Settings struct {
 	File          string                                  `json:"file"`
 	StdoutMaxSize int                                     `json:"stdoutMaxSize"`
 	Theme         Theme                                   `json:"theme"`
+	CacheMode     bool                                    `json:"cacheMode"` // for dsq at the moment
```

Comment:
Rather than cachemode I think it will be more precise if we call this "ForceNoImports" or something like that. Since it's specifically ONLY about not importing right now.

Reply:
Nevermind ignore this since there's more that  needs to change.